index.js

```diff
@@ -1,20 +1,345 @@
-const menuIcon = document.querySelector('#menu-icon')
-const menuToggle = document.querySelector('#menu-toggle')
-const menuSection = document.querySelector('#menu-section')
-
-menuIcon.addEventListener('click', ()=>{
-  menuSection.classList.add('menu-section-active')
-  document.body.style.overflowY = 'hidden';
-} )
-
-menuToggle.addEventListener('click', ()=>{
-  menuSection.classList.remove('menu-section-active')
-  document.body.style.overflowY = 'scroll';
-})
-
-document.querySelectorAll('.menu-item a').forEach((item)=> {
-  item.addEventListener('click', ()=>{
-    menuSection.classList.remove('menu-section-active')
-    document.body.style.overflowY = 'scroll';
-  })
-})
+const showCaseProject = {
+  title: "Multi-Post Stories",
+  description:
+    "A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standar dummy text.",
+  image: "./img/tonic2.png",
+  technologies: ["css", "html", "bootstrap", "Ruby"],
+  seeLive: "#",
+  seeSource: "#"
+};
+
+
+
+const projectsData = [
+  {
+    titleTop: "Professional art",
+    titleBottom: "printing data",
+    description:
+      "A daily selection of privately personalized reads; no accounts of sign-ups required. has been the industry's standard",
+    image: "./img/art1.png",
+    technologies: ["css", "html", "bootstrap"],
+    seeLive: "#",
+    seeSource: "#"
+  },
```

Comment:
- [ ] Let's kindly make sure the design of our popup strictly follows that of Figma(e.g., font, colors, images, text, margins). Let's check out the two screenshots below and make the necessary adjustments. See live, see source, technologies buttons are not showing up. Let the popup be populated with all the data from the project array. Please kindly fix the issues.

| Yours | Figma |
| --- | --- |
| ![yourpopup](https://user-images.githubusercontent.com/63087491/218251336-bf6d7850-06c8-41c9-a99c-83ce57eb4c75.PNG) | ![Capture](https://user-images.githubusercontent.com/63087491/218251343-7ea118a0-67e8-4635-a36f-353510059e2d.PNG) |

- [ ] Also, kindly check and compare your mobile design to that of Figma.